site/content/blogs/release-v123.en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'AWS Copilot v1.23: App Runner private service, Aurora Serverless v2 and more!'
+twitter_title: 'AWS Copilot v1.23'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.23: App Runner private service, Aurora Serverless v2 and more!
+
+Posted On: Oct 31, 2022
+
+The AWS Copilot core team is announcing the Copilot v1.23 release.   
+Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and has over 300 people online and nearly 2.5k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.23 brings several new features and improvements:
+
+- **App Runner private services**: [See detailed section](#app-runner-private-services).
+- **Support Aurora Serverless v2 in storage init**: [See detailed section](#support-aurora-serverless-v2-in-storage-init).
+- **Moving misplaced `http` fields in environment manifest (backward-compatible!):** [See detailed section](#moving-misplaced-http-fields-in-environment-manifest-backward-compatible).
+- **Enable VPC flow logs for environments:** [See detailed section](#enable-VPC-flow-logs-for-environments).
```

Comment:
Not super sure if we need a detailed section for this. What do you think?

Reply:
Yeah I think it is a small one I can jus remove it.